Sources/NIO/BaseSocket.swift

```diff
@@ -244,8 +251,8 @@ class BaseSocket: Selectable, BaseSocketProtocol {
         try addr.withMutableSockAddr { addressPtr, size in
             var size = socklen_t(size)
 
-            try withUnsafeFileDescriptor { fd in
-                try body(fd, addressPtr, &size)
+            try withUnsafeHandle {
```

Comment:
missing `self.`

Reply:
Just want to point out, not my fault!  The code was pre-existing in that state.  But, sure, happy to adjust that while Im here.

Reply:
oh sorry, indeed. Much appreciated if you could fix it :)